airbyte-webapp/src/packages/cloud/components/experiments/FreeConnectorProgram/FreeConnectorProgramEnrollmentButton/MailSvg.tsx

```diff
@@ -0,0 +1,26 @@
+export const MailSVG = () => {
```

Comment:
I think we lean towards using the actual .svg file rather than exporting it in a .tsx in the codebase

(can use `import { ReactComponent as WhateverYouWantToCallIt } from ....` to use it as a component)

Reply:
For sure, that makes sense. I'll keep it consistent with the rest of our SVGs!

Reply:
https://github.com/airbytehq/airbyte/pull/21396/commits/4e4778fe89823d43525fe164786903bbc02e3f53

Reply:
I'm not sure if there's something weird going on with the diff, but it looks like there may be a .tsx and a .svg for each image now?